### ISSUE TITLE
Issues when iteratnig over simple CSV file

### ISSUE THREAD

**Issue by octocode686** [deeplearning4j/deeplearning4j#6625]
I am receiving the following error and stack trace when I try to iterate over the attached CSV file. The code follows below:
xception in thread "ADSI prefetch thread" java.lang.RuntimeException: 
	at org.deeplearning4j.datasets.iterator.AsyncDataSetIterator$AsyncPrefetchThread.run(AsyncDataSetIterator.java:444)
Caused by: java.lang.NumberFormatException: For input string: "erca de cuatro millon de desplazado ..
	at sun.misc.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2043)
	at sun.misc.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
	at java.lang.Double.parseDouble(Double.java:538)
	at org.datavec.api.writable.Text.toDouble(Text.java:596)
	at org.datavec.api.util.ndarray.RecordConverter.toMinibatchArray(RecordConverter.java:181)
	at org.deeplearning4j.datasets.datavec.RecordReaderMultiDataSetIterator.next(RecordReaderMultiDataSetIterator.java:160)
	at org.deeplearning4j.datasets.datavec.RecordReaderDataSetIterator.next(RecordReaderDataSetIterator.java:366)
	at org.deeplearning4j.datasets.datavec.RecordReaderDataSetIterator.next(RecordReaderDataSetIterator.java:457)
	at org.deeplearning4j.datasets.datavec.RecordReaderDataSetIterator.next(RecordReaderDataSetIterator.java:86)
	at org.deeplearning4j.datasets.iterator.AsyncDataSetIterator$AsyncPrefetchThread.run(AsyncDataSetIterator.java:418)

Code:
  public static void main(String[] args) throws Exception {
        if(WORD_VECTORS_PATH.startsWith("/PATH/TO/YOUR/VECTORS/")) {
            throw new RuntimeException("Please set the WORD_VECTORS_PATH before running this example");
        }

        int batchSize = 64;     //Number of examples in each minibatch
        int vectorSize = 200;   //Size of the word vectors. 300 in the Google News model
        int nEpochs = 1;        //Number of epochs (full passes of training data) to train on
        int truncateReviewsToLength = 256;  //Truncate reviews with length (# words) greater than this
        final int seed = 0;     //Seed for reproducibility

        Nd4j.getMemoryManager().setAutoGcWindow(10000);  //https://deeplearning4j.org/workspaces

        //Set up network configuration
        MultiLayerConfiguration conf = new NeuralNetConfiguration.Builder()
                .seed(12345)
                .updater(new Adam(2e-2))
                .activation(Activation.LEAKYRELU)
                .weightInit(WeightInit.XAVIER)
                .l2(1e-4)
                .list()
                .layer(0, new DenseLayer.Builder().nIn(28 * 28).nOut(500).build())
                .layer(1, new DenseLayer.Builder().nIn(500).nOut(100).build())
                .layer(2, new OutputLayer.Builder(LossFunctions.LossFunction.RECONSTRUCTION_CROSSENTROPY)
                    .activation(Activation.SOFTMAX).nIn(100).nOut(10).build())
                .pretrain(false).backprop(true)
    .build();

        MultiLayerNetwork net = new MultiLayerNetwork(conf);
        net.init();
        net.setListeners(new ScoreIterationListener(1));

        //DataSetIterators for training and testing respectively
        WordVectors wordVectors = WordVectorSerializer.loadStaticModel(new File(WORD_VECTORS_PATH));
        //WordVectors bigVectors = WordVectorSerializer.loadStaticModel(new File (LONG_VECTORS_PATH));
        //SentimentExampleIterator train = new SentimentExampleIterator(DATA_PATH, wordVectors, batchSize, truncateReviewsToLength, true);
       // SentimentExampleIterator test = new SentimentExampleIterator(DATA_PATH, wordVectors, batchSize, truncateReviewsToLength, false);
        RecordReader rrTest = new CSVRecordReader(1);
        rrTest.initialize(new FileSplit(new File(DATA_PATH)));  
        DataSetIterator testIter = new RecordReaderDataSetIterator(rrTest, 150, 2, 2);
        // System.out.println(wordVectors.getWordVector(wordVectors.vocab().wordAtIndex(0)).length);
        //System.out.println(bigVectors.getWordVector(bigVectors.vocab().wordAtIndex(0)).length);
        System.out.println("Starting training");
        for (int i = 0; i < nEpochs; i++) {
            net.fit(testIter);
            testIter.reset();
            System.out.println("Epoch " + i + " complete. Starting evaluation:");
[Spanish_CLASS_annotated1-stemmed.zip](https://github.com/deeplearning4j/deeplearning4j/files/2501809/Spanish_CLASS_annotated1-stemmed.zip)

**Comment by octocode686** [comment]
Hi Alex,
I got it figured out and should have let you know earlier on, sorry about that.
All the best,
